bundles/org.openhab.ui/web/src/js/app.js

```diff
@@ -58,6 +58,11 @@ Vue.use(Trend)
 // Import Fulscreen Plugin
 import fullscreen from 'vue-fullscreen'
 Vue.use(fullscreen)
+// Import Vue Draggable Resizable
+import VueDraggableResizable from 'vue-draggable-resizable'
```

Comment:
According to RelativeCI it's only a ~70kB overhead on the initial JS (or +4%), so while it's not that much, couldn't this be lazy loaded with the `oh-plan-layout` (see how the `oh-grid-layout` component imports the `vue-grid-layout` library)?